src/boot/boot.js

```diff
@@ -74,9 +81,9 @@ function polyfillBundles(needed) {
  * This triggers loading of the necessary resources for the client
  *
  * @param {Document} doc
- * @param {Config} config
+ * @param {AnnotatorConfig} config
  */
-function bootHypothesisClient(doc, config) {
+export function bootHypothesisClient(doc, config) {
```

Comment:
FWIW, exporting this and `bootSidebarApp` and invoking them from `boot/index` helps `boot/index` feel more intuitive. I approve overall!